conference/cart.py

```diff
@@ -267,6 +269,18 @@ def send_order_confirmation_email(order: Order, current_site) -> None:
     )
 
 
+def handle_early_bird_ticket_limit():
```

Comment:
I like the idea, I'm just wondering if it might be even simpler – calling this handler on step2's POST, and instead of disabling in db it would just throw an error that early birds are gone already. WDYT?

Reply:
Yeah, that sounds like an even better idea.